### ISSUE TITLE
Track / log python errors

### ISSUE THREAD

**Issue by Flustered** [arribada/horizon-gui#94]
We should find a way or logging errors from the python process to help us debug.

**Comment by Flustered** [comment]
Via SSH I tried killing the process so I could re-start it on command line, but it restarts itself!

$ sudo pgrep python
$ sudo kill <pid>

We should start the process and write to a known logfile so we can run a script to list the last 20 entries...

**Comment by al2k** [comment]
Yup, flask is running as a service so it just auto starts again if it fails.

The way I used was to first stop the service:

systemctl stop startflask

Then you can run it manually from the command line and it will log to terminal atleast
